commons/commons-bio/src/main/kotlin/ebi/ac/uk/model/constants/ProcessingStatus.kt

```diff
@@ -0,0 +1,19 @@
+package ebi.ac.uk.model.constants
+
+import ebi.ac.uk.errors.UnsupportedStatusException
+
+private const val PROCESSED_STATUS = "PROCESSED"
+private const val PROCESSING_STATUS = "PROCESSING"
+
+sealed class ProcessingStatus(val value: String) {
+    companion object {
+        fun valueOf(value: String): ProcessingStatus = when (value) {
+            PROCESSED_STATUS -> Processed
+            PROCESSING_STATUS -> Processing
+            else -> throw UnsupportedStatusException(value)
```

Comment:
status comes from DB I think we should not handle invalid status as the system itself  is the only one that store value there is not that there is an api where user can set anything.

Reply:
Done